Source/KerbalCombatSystems/RocketTargeting.cs

```diff
@@ -150,16 +152,19 @@ public float Calculate()
             if (engines.Count < 1)
                 return -1;
 
-            float cosineLosses;
```

Comment:
Rockets are usually placed at slight angle for spin which creates cosine losses. The predicted acceleration will be off for the whole simulation without accounting for cosine losses.

Reply:
If possible: review the generic utility to highlight where averaging the direction and magnitude of the forward thrust won't properly account for the cosine losses.

Reply:
Total thrust for the same rocket over five updates:

Current:
100.7655
100.7653
100.7654
100.7655
100.7655

Suggested:
100.659
100.6579
100.6593
100.66
100.66

Not exactly the same but they all still hit and the new method accounts for engines with multiple thrust transforms so I'm happy to use it.

Reply:
Glad of it, utility based means we can apply it for other purposes